x/group/client/testutil/tx.go

```diff
@@ -2306,7 +2205,194 @@ func (s *IntegrationTestSuite) TestTxLeaveGroup() {
 	}
 }
 
-func (s *IntegrationTestSuite) getGroupIdFromTxResponse(txResp sdk.TxResponse) string {
+func (s *IntegrationTestSuite) TestExecProposalsWhenMemberLeavesOrIsUpdated() {
+	val := s.network.Validators[0]
+	clientCtx := val.ClientCtx
+
+	weights := []string{"1", "1", "2"}
+	accounts := s.createAccounts(3)
+	testCases := []struct {
+		name      string
+		votes     []string
+		members   []string
+		malleate  func(groupID string) (*cobra.Command, []string)
```

Comment:
```suggestion
		malleate  func(groupID string) err
```

my small preference would be to run the cobra command directly inside the malleate function, but small nitpick, no big deal if we keep as-is.